### DISCUSSION TITLE
How force set en(us) keyboard layout on login screen?

### DISCUSSION THREAD

**Discussion by x09** [neutrinolabs/xrdp#3184] [Q&A]
I want the English keyboard layout to always be enabled, no matter what I have active.

![изображение](https://github.com/user-attachments/assets/7369b556-c609-4366-b3f1-ffbec95549be)

I dont want any non-latin symbols here.

**Comment by matt335672** [comment]
Have a look in `/etc/xrdp/xrdp.ini`

There are some commented out debug settings which might do what you want:-

```
; The following options will override the keyboard layout settings.
; These options are for DEBUG and are not recommended for regular use.
#xrdp.override_keyboard_type=0x04
#xrdp.override_keyboard_subtype=0x01
#xrdp.override_keylayout=0x00000409
```
